docs/extensions.rst

```diff
@@ -616,21 +616,42 @@ the flag for the ``putup`` command (``--awesome-files``).
 Final Considerations
 ====================
 
-When writing extensions, it is important to be consistent with the default
-PyScaffold behavior. In particular, PyScaffold uses a ``pretend`` option to
-indicate when the actions should not run but instead just indicate the
-expected results to the user, that **MUST** be respected.
-
-The ``pretend`` option is automatically observed for files registered in
-the project structure representation, but complex actions may require
-specialized coding. The :mod:`~pyscaffold.log` module provides a
-special :class:`logger <pyscaffold.log.ReportLogger>` object useful in
-these situations. Please refer to `pyscaffoldext-cookiecutter`_ for a
-practical example.
-
-Other options that should be considered are the ``update`` and ``force``
-flags. See :obj:`pyscaffold.api.create_project` for a list of available
-options.
+#. When writing extensions, it is important to be consistent with the default
+   PyScaffold behavior. In particular, PyScaffold uses a ``pretend`` option to
+   indicate when the actions should not run but instead just indicate the
+   expected results to the user, that **MUST** be respected.
+
+   The ``pretend`` option is automatically observed for files registered in
+   the project structure representation, but complex actions may require
+   specialized coding. The :mod:`~pyscaffold.log` module provides a
+   special :class:`logger <pyscaffold.log.ReportLogger>` object useful in
+   these situations. Please refer to `pyscaffoldext-cookiecutter`_ for a
+   practical example.
+
+   Other options that should be considered are the ``update`` and ``force``
+   flags. See :obj:`pyscaffold.api.create_project` for a list of available
+   options.
+
+#. Don't forget that packages can be created inside namespaces.
+   To be on the safe side when writing templates prefer `explicit relative
+   import statements`_ (e.g. ``from . import module``) or use the template
+   variable ``${qual_pkg}`` provided by PyScaffold. This variable contains the
+   fully qualified package name, including eventual namespaces.
```

Comment:
including *possible* namespaces maybe? The word eventual is not used in the sense of potential or possible anymore as far as I know. But then again I am german and always had quite bad grades in English ;-)